CMakeLists.txt

```diff
@@ -84,13 +84,14 @@ make_directory(${CMAKE_BINARY_DIR}/locale)
 make_directory(${CMAKE_BINARY_DIR}/logs)
 
 # Copy ini files on first build
```

Comment:
Comment needs to be updated since it moves more than just ini file now

Reply:
Changed the comment